src/Scene/NodeProcess.js

```diff
@@ -217,6 +226,39 @@ function findAncestorWithValidTextureForLayer(node, layerType, layer) {
     }
 }
 
+const nullMesh = new THREE.Mesh();
+
+function updateNodeFeature(quadtree, node, layersConfig) {
+    const featureLayers = layersConfig.getGeometryLayers();
+    const promises = [];
+    for (let i = 0; i < featureLayers.length; i++) {
+        const layer = featureLayers[i];
+        const protocol = layer.protocol;
+        if (protocol.toLowerCase() == 'wfs') {
+            if (layer.tileInsideLimit(node, layer) && !node.content.get(layer.id)) {
+                const args = {
+                    layer,
+                };
+                const promise = quadtree.interCommand.request(args, node, refinementCommandCancellationFn).then((result) => {
+                    // if request return empty json, WFS_Provider.getFeatures return undefined
+                    if (result.feature) {
+                        result.feature.layer.root.add(result.feature);
```

Comment:
Note: you add everything to the root node of the layer; that's just another example of why Layer shouldn't have a hierarchical (inheritance from Node).

I'm not convinced either by the duplication: storing `result.feature` twice looks suspicious...